test/module/libs/crypto/signature_test.cpp

```diff
@@ -44,78 +44,3 @@ TEST(Signature, PrintkeyPair) {
   ASSERT_NO_THROW({ std::cout << keypair.pubkey.to_base64() << std::endl; });
   ASSERT_NO_THROW({ std::cout << keypair.privkey.to_base64() << std::endl; });
 }
-
-TEST(Signature, generatedByAndroid) {
```

Comment:
Better to add a signature test for cryptography. We should add tests for crypto or task about it.

Reply:
Currently there is test for both sign and verify

https://github.com/hyperledger/iroha/blob/17af47bdda507c7999670699a9f1abc089887b59/test/module/libs/crypto/signature_test.cpp#L30

Additional tests can be added in another task.